### ISSUE TITLE
Button Component Missing Accessibility Attributes

### ISSUE THREAD

**Issue by ankitrraj** [DjedAlliance/StablePay-LandingPage#43]
Button component is missing type="button" attribute and doesn't accept onClick or other HTML button props. This affects accessibility and reusability.
